README.md

```diff
@@ -11,11 +11,34 @@ AlertsViewer is an internal project helping us with myriad alert-related needs.
 
 [AlertsViewer application architecture](ARCHITECTURE.md)
 
+## Setup
+
+### Requirements
+
+- MBTA API key (get one [here](https://api-dev.mbtace.com))
+  - **Note:** This key must have its version set to `2021-01-09`
```

Comment:
question: Is this something you came across in practice? It's possible that since we aren't doing anything very specific so any version would work, but I don't actually know.

Reply:
No, I just copied what they have over in the concierge >_> I'll just take it out.